package.json

```diff
@@ -12,6 +12,7 @@
     "patch-release": "npm version patch && npm publish && git push --follow-tags",
     "minor-release": "npm version minor && npm publish && git push --follow-tags",
     "major-release": "npm version major && npm publish && git push --follow-tags",
+    "release": "release-it",
```

Comment:
Cool tool! Didn't know this existed. If you could, please rewrite the other `-release` scripts to use `release-it` with its various options. Otherwise just remove `release-it` and I'll implement on my own.